primeqa/qg/run_qg.py

```diff
@@ -38,7 +39,7 @@ class ModelArguments:
         default="table",
         metadata={
             "help": "Whether to generate questions from tables or passages",
-            "choices": ["table", "passage"],
+            "choices": ["table", "passage_qg", "passage_qa2s"],
```

Comment:
The Primeqa QG only supports the modalities 'passage,' 'table,' and 'hybrid.' It also makes no sense to include modality options that do not correlate with its meaning.

Reply:
I've introduced a new parameter `gen_config` in order to separate the method for the generation from the modality.